app/exporters/drug_stocks_report_exporter.rb

```diff
@@ -33,17 +33,20 @@ def drug_categories_header
     left_padding_columns = [nil] * left_pad_size
 
     left_padding_columns + @drugs_by_category.flat_map do |category, drugs|
-      [protocol_drug_labels[category][:full], [nil] * drugs.count].flatten
+      category_column_count = show_patient_days?(category) ? drugs.count : drugs.count - 1
```

Comment:
Something might be up with this calculation, `Column M` should say diabetes
![image](https://user-images.githubusercontent.com/16774200/196627340-212fde76-54ed-4cf5-8e24-602e63b4fdbd.png)

Reply:
@prabhanshuguptagit where'd you see this? Local test data? Is Metformin set as a diuretic drug?

Reply:
Oh whoops. I think I was going back and forth changing Metmorfin's type on local test data yes. Not a bug.

<img width="574" alt="image" src="https://user-images.githubusercontent.com/16774200/196858107-03c9d5ac-1e86-4efa-b93b-adc0dd78fb4a.png">